src/css/input.css

```diff
@@ -118,6 +123,36 @@ article a[href*="//"]:not([href*="defichainwiki.com"])::after
   content: "" !important;
 }
 
+.formula_bubble {
```

Comment:
Where is this needed?

Reply:
Nowhere, it was used in one of the version of solving backgrounds for numeric examples.  It should be removed.